### PR TITLE
Fixed: Random Survivor Skin onNewLogin

### PR DESCRIPTION
**Pull request by HeroSmurf**
- Re-Adds simple randomization for 2 Common Male Survivor Skins upon new
  character creation

--- Examples ---
- Survivor1_DZ       http://goo.gl/d7nGkt
- Survivor2_DZ       http://goo.gl/KD8z0g
- player_monitor.fsm always passes "Survivor2_DZ" when Male gender is selected. We check for this being passed and then re-assign randomly.

### CODE REVIEW

**Comment by R4Z0R49** [comment]
Lol thought we fixed this last patch.

Thanks

**Comment by KPanicEKS** [comment]
Are the survivor skins server side now? If so, could we perhaps add the
different survivor skins to the the loot tables similar to the
camo, solider and ghillie? Not sure how that would work if a bandit tried
wearing one but might be worth exploring.

**Comment by HeroSmurf** [comment]
Sure. 

M0tzilla do you mind if that's a later pull and we get this fairly simple one merged in here for now?
I'd like to get it out for 1.8.6 if possible. 

Your request is a little more complicated and I'd like to test it (and not hold up or miss this update for 1.8.6 etc)

**Comment by KPanicEKS** [comment]
hell ya it can, really looking forward to 1.8.6.

thanks!

**Comment by HeroSmurf** [comment]
Here's a follow up

You're right about skins being client side. Foxy told me the following: 
Survivor2 is actually a PMC lite skin with a client side randomization routine that was turned off. 

So 1.8.6 will see no change for the number of survivor skins.

**Comment by bchawke** [comment]
ELI5 (okay maybe ELI15) why we used to have different survivor skins but now there's only one.  I never really understood what changed that.

**Comment by AsFoxy** [comment]
There was only ever one skin but with two (or three?) different textures, one of which was selected randomly for each client every time they logged in. This meant each player would see a different skin and it would change for them every time they relogged.

**Comment by bchawke** [comment]
Yes, we would see different textures as they were randomly chosen, the part I don't understand is why that used to happen but now we all only see one survivor texture.

thanks
